src/object-store/src/lib.rs

```diff
@@ -19,7 +19,9 @@ pub use opendal::{
     Operator as ObjectStore, Reader, Result, Writer,
 };
 
+pub mod error;
 pub mod layers;
 mod metrics;
+pub mod object_store_manager;
```

Comment:
suggestion: `manager` should be enough.
```suggestion
pub mod manager;
```